src/main/java/core/basesyntax/CalculateArea.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface CalculateArea {

```

Comment:
Don’t use verbs for class/interface names
Bad example:
```
public interface CalculateArea {
}
```
Improved example:
```
public interface AreaCalculator {
}
```

Reply:
done!